runelite-api/src/main/java/net/runelite/api/widgets/WidgetInfo.java

```diff
@@ -153,6 +153,8 @@
 	BANK_SETTINGS_BUTTON(WidgetID.BANK_GROUP_ID, WidgetID.Bank.SETTINGS_BUTTON),
 	BANK_TUTORIAL_BUTTON(WidgetID.BANK_GROUP_ID, WidgetID.Bank.TUTORIAL_BUTTON),
 
+	GROUP_STORAGE_CONTAINER(WidgetID.GROUP_STORAGE_GROUP_ID, WidgetID.GroupStorage.CONTAINER),
```

Comment:
This is unused and incorrect. 724.45 appears to be the "All" button for the quantity selector. Remove it and its associated WidgetID definition.